### PR TITLE
taxonomy: data quality translations

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: data quality translations

### CODE REVIEW

File: taxonomies/data_quality.txt [hunk 1]
```diff
@@ -980,8 +1016,33 @@ description:en:Fibers are mentioned on nutrition photos, but are not present in
 
 ## Ingredients
 
+<<<<<<< Updated upstream
+=======
```
Comment (teolemon) [inline taxonomies/data_quality.txt:1020]:
```suggestion
```

File: taxonomies/data_quality.txt [hunk 2]
```diff
@@ -980,8 +1016,33 @@ description:en:Fibers are mentioned on nutrition photos, but are not present in
...
+en:nutri-score-grade-from-category-does-not-match-calculated-grade
+description:en: The calculated nutriscore grade is different than the one expected for this category
+fix_action:en: add_nutrition_facts
+
+<en:Ingredients errors
+en:ingredients-single-ingredient-from-category-does-not-match-actual-ingredients
+description:en: This category expect a single and specific ingredient
+fix_action:en: add_ingredients_text
+
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/data_quality.txt:1040]:
```suggestion

```

File: taxonomies/data_quality.txt [hunk 3]
```diff
@@ -2731,12 +2859,23 @@ description:en:This might be due among other things to incorrect nutrition value
 en:Nutrition 3 or more values are identical
 description:en:Someone may have entered wrong data because there are 3 identical values in nutrition table and those values are above 1
 
+<<<<<<< Updated upstream
+=======
```
Comment (teolemon) [inline taxonomies/data_quality.txt:2863]:
```suggestion

```

File: taxonomies/data_quality.txt [hunk 4]
```diff
@@ -2731,12 +2859,23 @@ description:en:This might be due among other things to incorrect nutrition value
...
+<en:Nutrition warnings
+en:Nutrition all values zero
+es:Todas las valores de nutrición son cero
+fr:Toutes les valeurs nutritionnelles sont à zéro
+#description:en:
+
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/data_quality.txt:2870]:
```suggestion

```

File: taxonomies/data_quality.txt [hunk 5]
```diff
@@ -2761,42 +2900,71 @@ en:Nutrition value - total over 1000
...
 
 <en:Nutrition warnings
 en:Nutrition value under 0,01 g - Salt
+es:Valor nutricional inferior a 0,01 g - Sal
+fr:Valeur nutritionnelle inférieure à 0,01 g - Sel
+#description:en:
+
+<en:Nutrition warnings
+<<<<<<< Updated upstream
+=======
```
Comment (teolemon) [inline taxonomies/data_quality.txt:2915]:
```suggestion
<en:Nutrition warnings
```

File: taxonomies/data_quality.txt [hunk 6]
```diff
@@ -2761,42 +2900,71 @@ en:Nutrition value - total over 1000
...
+en:Nutrition value under 0,1 g salt
+es:Valor nutricional inferior a 0,1 g de sal
+fr:Valeur nutritionnelle inférieure à 0,1 g de sel
 #description:en:
 
 <en:Nutrition warnings
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/data_quality.txt:2922]:
```suggestion
```

File: taxonomies/data_quality.txt [hunk 7]
```diff
@@ -2912,6 +3085,21 @@ en:Carbon footprint from known ingredients more than from meat or fish
 
 ### Labels
 
+<<<<<<< Updated upstream
+=======
```
Comment (teolemon) [inline taxonomies/data_quality.txt:3089]:
```suggestion
```

File: taxonomies/data_quality.txt [hunk 8]
```diff
@@ -2912,6 +3085,21 @@ en:Carbon footprint from known ingredients more than from meat or fish
...
+es:etiqueta vegana pero ingrediente no vegano
+description:en:This product has a vegan label but contains non-vegan ingredients.
+
+<en:Data quality errors
+en:vegetarian-label-but-non-vegetarian-ingredient
+fr:étiquette végétarienne mais ingrédient non végétarien
+es:etiqueta vegetariana pero ingrediente no vegetariano
+description:en:This product has a vegetarian label but contains non-vegetarian ingredients.
+
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/data_quality.txt:3102]:
```suggestion

```

File: taxonomies/data_quality.txt [hunk 9]
```diff
@@ -2912,6 +3076,18 @@ en:Carbon footprint from known ingredients more than from meat or fish
...
+fr:étiquette végétalienne mais ingrédient non végétalien
+es:etiqueta vegana pero ingrediente no vegano
+description:en:This product has a vegan label but contains non-vegan ingredients.
+
+<en:Data quality errors
+en:vegetarian-label-but-non-vegetarian-ingredient
+fr:étiquette végétarienne mais ingrédient non végétarien
+es:etiqueta vegetariana pero ingrediente no vegetariano
+description:en:This product has a vegetarian label but contains non-vegetarian ingredients.
+
```
Comment (benbenben2) [inline taxonomies/data_quality.txt:3090]: Same as previous comment. This is error, it should be in the first part of the file around line 811

```suggestion
```

**Review by benbenben2 — CHANGES_REQUESTED** [review]
Almost perfect, just error and warning should not be mixed up. See comments.

**Review by benbenben2 — APPROVED** [review]
all good

**Pull request merged**